client/src/components/GridField/GridField.scss

```diff
@@ -55,6 +55,7 @@
   }
 
   &:after {
+    // equivalent of font-icon-caret-up-down
     content: ")";
     display: inline-block;
     font-family: "silverstripe";
```

Comment:
This will be handled separately as per https://github.com/silverstripe/silverstripe-admin/issues/1957#issuecomment-3225952066
For now just labelling which icon it is.